examples/classifier/CL1_compute_dtw_distance.py

```diff
@@ -0,0 +1,33 @@
+# Standard library imports
+import sys
+new_path = sys.path[0][:-len("/examples/timeseries")]
+sys.path.append(new_path)
+import warnings
+warnings.filterwarnings(("ignore"))
+
+# Third party imports
+# /
+
+# Local application imports
+from scifin import timeseries as ts
+from scifin import classifier as cl
```

Comment:
are you still importing everything from all modules when initialising? 

Such approach is not recommended and one should avoid doing so in general, since it create unnecessary memory storage and likely to introduce conflicts (e.g. different modules having same class/variable names)

Reply:
I have to change that habit indeed. Thanks for the tip!